posthog/api/cohort.py

```diff
@@ -172,6 +334,29 @@ class Group(BaseModel, extra="forbid"):
 
 class CohortFilters(BaseModel, extra="forbid"):
     properties: Group
+    realtimeSupported: bool = False
+
+    @model_validator(mode="after")
+    def _check_realtime_support(self):
+        """Check if all filters have valid bytecode to determine realtime support."""
+        self.realtimeSupported = self._all_filters_have_bytecode(self.properties)
```

Comment:
This should just indicate the type of the cohort - **not** a seperate model prop